client/state/application-passwords/schema.js

```diff
@@ -0,0 +1,13 @@
+export const itemsSchema = {
```

Comment:
This schema is completely duplicating [this part of the corresponding data layer schema ](https://github.com/Automattic/wp-calypso/blob/01dccfae03148121ff7820ef417929a12c64aec4/client/state/data-layer/wpcom/me/two-step/application-passwords/schema.js#L4-L15). I wonder if we should import it here, or keep them separate 🤔

Reply:
Sharing is probably fine and I would normally encourage it. In this case, I have some reservations just thinking about chunks, sections, webpack 4, etc. This is really small and simple, maybe best to just duplicate it.

Reply:
It also made sense to me, because it separates the endpoint response structure from the state tree structure (which will not necessarily be the same). Keeping as-is 👍